contracts/trader/TraderPoolInvestProposal.sol

```diff
@@ -349,6 +350,8 @@ contract TraderPoolInvestProposal is ITraderPoolInvestProposal, TraderPoolPropos
         );
 
         delete _proposalInfos[proposalId].newInvestedBase;
+
+        emit ProposalConvertToDividends(proposalId, msg.sender, newInvestedBase);
```

Comment:
If you are adding this event, lets remove the emission of `withdrawn` and `supplied` events. The graph has to be updated

Reply:
Also naming. Please rename to `ProposalConverted`